src/Html.php

```diff
@@ -2841,6 +2841,7 @@ public static function showDateField($name, $options = [])
             dateFormat: 'Y-m-d',
             wrap: true, // permits to have controls in addition to input (like clear or open date buttons
             weekNumbers: true,
+            time_24hr: true,
```

Comment:
@cconard96 Could 24h format be used for everyone, or will it be a problem for english users?

Reply:
Some people understand it and some people don't.
In the US, 12 hour format with AM and PM is the default and the only format widely taught. For European English speakers, I don't think 24 hour time would be an issue.

Reply:
> Some people understand it and some people don't. In the US, 12 hour format with AM and PM is the default and the only format widely taught. For European English speakers, I don't think 24 hour time would be an issue.

I think we can merge this one. As I said in initial issue:
> GLPI has no configuration that gives ability to the user to change between `12 am/pm` and `24 hours` format for time.
> So forcing `time_24hr` value on flatpickr initialization seems indeed an easy solution to make time selection consistent with time displaying.

Maybe we could introduce a 12h/24h format preference in main branch, but it will be another PR.